hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/embedded/EmbeddedTimelineServerHelper.java

```diff
@@ -50,10 +50,13 @@
       if (!TIMELINE_SERVER.isPresent() || !TIMELINE_SERVER.get().canReuseFor(config.getBasePath())) {
         TIMELINE_SERVER = Option.of(startTimelineService(context, config));
       }
+      updateWriteConfigWithTimelineServer(TIMELINE_SERVER.get(), config);
```

Comment:
Add a else branch and do not modify the startTimelineService method.

Reply:
ok, I will modify this